rl_zoo3/record_video.py

```diff
@@ -153,8 +153,8 @@
                 episode_start=episode_starts,
                 deterministic=deterministic,
             )
-            obs, _, dones, _ = env.step(action)
-            episode_starts = dones
+            obs, _, terminated, truncated, _ = env.step(action)
+            episode_starts = np.logical_or(terminated, truncated)
```

Comment:
Following https://github.com/DLR-RM/rl-baselines3-zoo/pull/256#issuecomment-1291891924

This should be reverted, right? Is there others?

Reply:
yes, everytime the env is a vec env, I think the only other might be the enjoy script.